src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -51,6 +59,39 @@
       fullNames.add(namer.getProtoFileImportFromService(targetInterface));
     }
 
+    return createImportFilesFromNames(fullNames);
+  }
+
+  public List<ImportTypeView> generateProtoFileImports(SurfaceTransformerContext context) {
```

Comment:
Will the bfsHandler load all the proto classes in a BFS traversal? If so, it might not be necessary, since xxx_pb.rb (such as pubsub_pb.rb will load the proto classes it depends on)